chain/test/Test/Pos/Chain/Update/Arbitrary.hs

```diff
@@ -44,6 +45,10 @@ instance Arbitrary BlockVersionModifier where
     arbitrary = genericArbitrary
     shrink = genericShrink
 
+instance Arbitrary ConsensusEra where
+    arbitrary = genericArbitrary
+    shrink = genericShrink
```

Comment:
I actually like the generator function, especially since what its generating is an enum with two values, which therefore does not have an interesting shrink  function.